mergJSON.lcdoc

```diff
@@ -0,0 +1,86 @@
+Library: mergJSON
+
+type: library
+
+Description:
+mergJSON is an external for encoding and decoding [JSON](http://json.org).
+
+This external has been implemented to encode/decode UTF8 JSON to a LiveCode array as quickly as possible. Currently externals can not work with multi-dimensional LiveCode arrays so the intention is it is used with two functions like this:
+
+    -- pArray - array to be encoded
+	  -- pForceRootType - can force the root to be an object if it looks like an array
+	  -- pPretty - include whitespace
+	  function ArrayToJSON pArray,pForceRootType,pPretty
+	    repeat for each key tKey in pArray
+	      if pArray[tKey] is an array then
+	         put "}"&ArrayToJSON(pArray[tKey]) into pArray[tKey]
+	      end if
+	    end repeat
+	    return(mergJSONEncode("pArray",pForceRootType,pPretty))
+	  end ArrayToJSON
+    
+	  function JSONToArray pJSON
+	    local tArray,tKeys
+	    repeat for each line tKey in mergJSONDecode(pJSON,"tArray")
+	      put JSONToArray(tArray[tKey]) into tArray[tKey]
+	    end repeat
+	    return tArray
+	  end JSONToArray
```

Comment:
I think you might get some weird indentation going on in this code block because you appear to have indented with "tab = 2 spaces" but Markdown assumes "tab = 4 spaces".